ur_robot_driver/resources/rtde_output_recipe.txt

```diff
@@ -22,3 +22,9 @@ tool_output_current
 tool_temperature
 robot_mode
 safety_mode
+output_double_register_24
+output_double_register_25
+output_double_register_26
+output_double_register_27
+output_double_register_28
+output_double_register_29
```

Comment:
Note that you wrote the torque data into registers 4-9, not 24-29. It might make sense to read those instead.

Reply:
@fmauch I implemented the changes above on my local `rtde_output_recipe.txt` but the error "Could not get fresh data package from robot" still appeared. After looking through URscript documentation and software requirements, I found that for e-series, the software version must be at least 5.3 to access registers 24..47. However, my robot is at 5.3.1 and I still see this error. If I keep the register index below 24, no errors show up.

Instead of outputting effort data to `output_double_register_X`, if I want to incorporate effort data into `JointState` message, how can I do that?